src/Playwright/Transport/StdIOTransport.cs

```diff
@@ -79,6 +79,7 @@ public void Close(string closeReason)
                 _readerCancellationSource?.Cancel();
                 try
                 {
+                    _process?.Dispose();
```

Comment:
Shouldn't `Kill` be called before `Dispose`?

Reply:
Doesn't make a difference. It still does what it needs to do and close the node.exe instead of leaving it orphan in the processes causing memory leaks.

Reply:
IMO it makes a huge difference, calling methods on disposed object is not a good practice.

Please refer to https://docs.microsoft.com/en-us/dotnet/api/system.idisposable?view=net-6.0#remarks